packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -20,12 +26,17 @@ export type AllowedTypes = readonly LazyItem<TreeNodeSchema>[];
 type AllowedTypesUnsafe = readonly LazyItem<TreeNodeSchemaUnsafe>[];
 
 // @public
-type ApplyKind<T, Kind extends FieldKind, DefaultsAreOptional extends boolean> = {
+type ApplyKind<T, Kind extends FieldKind> = {
```

Comment:
This is a breaking change, no?

Reply:
This is a system type. Would be real nice if that was shown in the API report. Can you fix that?

Reply:
I have a note to file a feature request. That's as far as I've gotten. But I suspect this would be an easy feature to get in.

Reply:
I may spend some time this afternoon filing tickets and repros with them. It's been a while...